### ISSUE TITLE
Update demos to include the link label for current pages

### ISSUE THREAD

**Issue by notlee** [Financial-Times/o-header#402]
We set a "current page" aria label in our demos for the current page but this removes context for screen reader users. Instead we should also include the link label. 

i.e. `aria-label="Current page" aria-current="page"` should be `aria-label="{{label}}, current page" aria-current="page"`

ft.com already do this

**Comment by notlee** [comment]
Today I tried to close one bug and uncovered a few more
https://github.com/Financial-Times/origami-navigation-service/pull/205
